solutions/quickstart/DA-types.md

```diff
@@ -0,0 +1,35 @@
+## Configuring network profile input for VPC <a name="network-profile"></a>
+
+This variable lets you choose from predefined network security profiles that control the default **Network ACL behavior**, **public gateway availability**, and whether **default Security Group (SG) rules are cleaned**. It simplifies deployment by offering four security levels ranging from fully open to fully restricted.
+
+- **Variable name:** `network_profile`
+- **Type:** `string`
+- **Default value:** `"standard"`
+- **Allowed values:** `open`, `standard`, `ibm-internal`, `closed`
+
+The selected profile automatically defines:
+
+| Profile name        | Default Behavior                                                  | Public Gateway | Default SG Rules |
+|----------------|------------------------------------------------------------------|----------------|------------------|
+| `open`         | Allow all inbound and outbound traffic                           | **Enabled**    | **Preserved**    |
+| `standard`       | Allow SSH(22), HTTP(80), HTTPS(443) and IBM internal rules         | **Enabled**    | **Preserved**    |
+| `ibm-internal` | No customer inbound traffic, only IBM internal and VPC connectivity| **Disabled**   | **Cleaned**      |
```

Comment:
I thought we said in the last deep dive we should not call it `ibm-internal`. We said we should call it something that is consistent with the IBM Cloud private network docs. Something like  "IBM Cloud private backbone" ?